pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -516,7 +516,13 @@ func (p *MasqueradePodInterface) createBridge() error {
 }
 
 func (p *MasqueradePodInterface) createNatRules() error {
+	if Handler.UseIptables() == nil {
+		return p.createNatRulesUsingIptables()
+	}
+	return p.createNatRulesUsingNftable()
```

Comment:
should usage of nftable actually be the default, and iptables be the fallback?

Reply:
I don't think so. If iptables are enable in the host and we configure nftables it will not work ( it will let you configure the nftables but they will not work).

I wrote it this way so if we have an error in the iptables this mean the host is using nftables.